### PR TITLE
README organization

### PR DESCRIPTION
**Pull request by disabledandfab**
Edited README to better organize ports list via <details> blocks and emojis

### CODE REVIEW

**Comment by andreasgrafen** [comment]
Heya! Thank you for this, the Emojis are a great idea! <3

But just from a usability standpoint there are some issues with this. First and foremost: This is not searchable. When the details are closed – which they are by default – users can't `CTFL + F` to find their port. This introduces a lot of clicking. It could also be prone to user error as one might expect a port to be in one category, while it is actually in another one. :/

Aside from that: Is it intentional that a lot of categories are hidden underneath the `Leisure` one? Terminals for example don't seem to fit that category. Note-Taking apps are also not within their own category anymore due to markup errors. o:

File: README.md [hunk 1]
```diff
@@ -928,39 +932,44 @@ Catppuccin is available for various apps and in different formats. Here is a lis
 - [PyRadio](https://github.com/catppuccin/pyradio)
 - [Minecraft](https://github.com/catppuccin/minecraft)
```
Comment (andreasgrafen) [inline README.md:934]: Category is seemingly missing the closing </details> tag.

File: README.md [hunk 2]
```diff
@@ -928,39 +932,44 @@ Catppuccin is available for various apps and in different formats. Here is a lis
...
 
 - [Cinny](https://github.com/catppuccin/cinny)
 - [Discord](https://github.com/catppuccin/discord)
 - [Element](https://github.com/catppuccin/element)
 - [Revolt](https://github.com/catppuccin/revolt)
 - [Telegram](https://github.com/catppuccin/telegram)
+</details>
 
-#### Note-taking
+ <details><summary>📝Note-taking
```
Comment (andreasgrafen) [inline README.md:966]: Summary is missing the closing tag.

File: README.md [hunk 3]
```diff
@@ -915,8 +918,9 @@ Catppuccin is available for various apps and in different formats. Here is a lis
 - [ShareX](https://github.com/catppuccin/sharex)
 - [Sumatra PDF](https://github.com/catppuccin/sumatra-pdf)
 - [Zathura](https://github.com/catppuccin/zathura)
+</details>
 
-#### Leisure
+ <details><summary>🌱Leisure</summary>
```
Comment (andreasgrafen) [inline README.md:923]: Details is missing the closing tag.

File: README.md (hunk 2 above)
Comment (disabledandfab) [reply README.md:966]: Thank you for the advice. I'm attempting Hacktoberfest, but clearly  I've more work to do. Should I convert this to a draft or open a new request once I've improved it?

**Comment by andreasgrafen** [comment]
> When the details are closed – which they are by default […]

By the way! Details can be opened by default by just setting it as an attribute. While this doesn't improve the initiall wall of text, this could still be a good addition to allow the user to close categories?

```html
<details open>
  <summary>open by default</summary>
  content
</details>
```

**Comment by andreasgrafen** [comment]
Heya @disabledandfab! c:
You've marked this as ready for review, but there are no recent changes on the repo. Perhaps you forgot to push them?

**Comment by andreasgrafen** [comment]
As mentioned above; I really like the addition of the emojis. Makes it a lot more friendly – maybe they could benefit from a `space` between them and the text? c:
Regarding the `mark`s … on my end they def. don't get applied properly – do they for you and if so, could you provide a screenshot of how it's supposed to look?

!(Screenshot of Section Header not displaying the mark properly)[https://user-images.githubusercontent.com/35840154/194955939-8fc3591d-67d2-47fe-a1ac-8b48b82988a3.png]

**Comment by andreasgrafen** [comment]
Sorry for the late reply; life took over. It seems like the changes aren't on Github yet. :/

**Pull request merged**